src/coreclr/vm/jitinterface.cpp

```diff
@@ -10871,12 +10867,22 @@ void CEECodeGenInfo::getHelperFtn(CorInfoHelpFunc    ftnNum,               /* IN
 
```

Comment:
The current scheme where `hlpFuncTable` entry that is less than `DYNAMIC_CORINFO_HELP_COUNT` implies dynamic helper won't work reliably for wasm. Wasm function pointers are ordinals that start with like `1`, so this trick won't work.

For wasm, I think we need to change `hlpFuncTable` that has just native code pointers, and separate table that has the information about dynamic entry points (or change `hlpFuncTable` to be array of structs).